layers/+lang/ruby/README.org

```diff
@@ -164,6 +166,13 @@ When =ruby-test-runner= equals =ruby-test=.
 | ~SPC m t b~ | run test file       |
 | ~SPC m t t~ | run test at pointer |
 
+** seeing-is-believing
+
+| Key Binding     | Description                      |
+|-----------------+----------------------------------|
+| ~<SPC> m s b~   | Run seeing is believing          |
+| ~<SPC> m s c~   | Clear seeing is believing output |
+
```

Comment:
Need to find other key bindings, these one are reserved for REPL interactions.
Can those two functions be unified together into a toggle ?

Reply:
This ain't changing.

Reply:
I haven't used spacemacs in quite a while. Someone else is welcome to find better bindings, but I doubt I will ever do it.